src/elements/hx-progress/index.spec.js

```diff
@@ -0,0 +1,38 @@
+import { fixture, expect } from '@open-wc/testing';
+
+/**
+ * <hx-progress> component tests
+ *
+ * @type HXProgressElement
+ *
+ */
+describe('<hx-progress> tests', () => {
```

Comment:
Please add a screenshot of the test run in the `Description` block.

Reply:
Changes have been done, please review.